README.md

```diff
@@ -0,0 +1 @@
+LivelyHWTracker
```

Comment:
Hi @HolaFlight , can you squash the two commits into one before merging into main?